operations/system_configuration_service/Configure_BMC_and_Controller_Parameters_with_scsd.md

```diff
@@ -34,7 +34,12 @@ The NTP server and syslog server for BMCs in the liquid-cooled cabinet are typic
 
 ## Details
 
-Setting the SSH keys for mountain controllers is done by running the `/opt/cray/csm/scripts/admin_access/set_ssh_keys.py` script:
+### Prerequisites
+
+* The latest CSM documentation is installed on the system. See [Check for Latest Documentation](../../update_product_stream/index.md#check-for-latest-documentation).
+
+Setting the SSH keys for mountain controllers is done by running the `/usr/share/doc/csm/scripts/operations/configuration/set_ssh_keys.py` script:
+*NOTE*: The `set_ssh_keys.py` located in `/opt/cray/csm/scripts/admin_access` contains a bug and should not be used.
```

Comment:
This section should probably be moved to right before details. Something like the following:
```
## Prerequisites

* The latest CSM documentation is installed on the system. See [Check for Latest Documentation](../../update_product_stream/index.md#check-for-latest-documentation).

## Details
Setting the SSH keys for mountain controllers is done by running the `/usr/share/doc/csm/scripts/operations/configuration/set_ssh_keys.py` script:
*NOTE*: The `set_ssh_keys.py` located in `/opt/cray/csm/scripts/admin_access` contains a bug and should not be used.
```